bindings/dart/example/lib/main.dart

```diff
@@ -59,11 +57,8 @@ class _MyAppState extends State<MyApp> {
     super.initState();
 
     // mobile is always loaded with multi-threading features, web only optionally
-    final hint =
-        kIsWeb ? FeatureHint.webSequential : FeatureHint.mobileParallel;
-
     debugPrint('start loading assets');
-    Logic.load(rootBundle, hint).then((logic) {
+    Logic.load(rootBundle, features: []).then((logic) {
```

Comment:
Since `[]` is the default we can omit it, or there is a reason why you want to make this explicit?

Reply:
it's more or less a leftover from testing. i'll remove it for now. once the web parallel feature is stabilized, we can set it as the default in the example app.